### PR TITLE
Pa main menu

### PR DESCRIPTION
**Pull request by punamahire**
Created Post Manager and Post Repository to show post options on console.

### CODE REVIEW

**Review by acheatham87 — APPROVED** [review]
:shipit:

File: TabloidCLI/UserInterfaceManagers/PostManager.cs [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            string url = Console.ReadLine();
+            if (!string.IsNullOrWhiteSpace(url))
+            {
+                postToEdit.Url = url;
+            }
+            Console.Write("New Publish DateTime (blank to leave unchanged: ");
+            string dateTime = Console.ReadLine();
+            if (!string.IsNullOrWhiteSpace(dateTime))
+            {
+                //postToEdit.PublishDateTime = dateTime;
```
Comment (DougSilvera) [inline TabloidCLI/UserInterfaceManagers/PostManager.cs:141]: is this supposed to be commented out or active code?

Comment (punamahire) [reply TabloidCLI/UserInterfaceManagers/PostManager.cs:141]: In the future tickets I will take care of it. It is just a placeholder for now.

Comment (DougSilvera) [reply TabloidCLI/UserInterfaceManagers/PostManager.cs:141]: perfect! just checking before i rubber stamp it thanks!

**Review by DougSilvera — APPROVED** [review]
:shipit:

**Review by LukeSlater02 — COMMENTED** [review]
💯

**Review by LukeSlater02 — APPROVED** [review]
:shipit:

**Pull request merged**